doc/ref_cert/lfn/chapters/chapter04.md

```diff
@@ -86,7 +86,44 @@ It covers the test cases against image management operations.
 
 
 #### 4.3.4.3 Block Storage - Cinder
-It covers the test cases against volume management operations.
+
+Cinder API is covered in the OpenStack Gates via
+[Tempest](https://opendev.org/openstack/tempest) and
+[cinder-tempest-plugin](https://opendev.org/openstack/cinder-tempest-plugin)
+as integrated in
+[Functest Smoke CNTT](https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml).
+
+According to
+[RA1 Core OpenStack Services APIs](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md)
+the next test names must not be executed:
+
+| test rejection regular expressions                                                                   | reasons                               |
+|------------------------------------------------------------------------------------------------------|---------------------------------------|
+| .\*test_incremental_backup                                                                           | https://gerrit.opnfv.org/gerrit/68881 |
+| .\*test_consistencygroups                                                                            | consistency_group                     |
+| .\*test_group_snapshots.GroupSnapshotsV319Test.test_reset_group_snapshot_status                      | https://launchpad.net/bugs/1770179    |
+| .\*test_multi_backend                                                                                | multi-backend                         |
+| .\*test_volume_retype.VolumeRetypeWithMigrationTest                                                  | multi-backend                         |
+| .\*test_volume_delete_cascade.VolumesDeleteCascade.test_volume_from_snapshot_cascade_delete          | https://launchpad.net/bugs/1677525    |
+| .\*test_volumes_backup.VolumesBackupsTest.test_volume_backup_create_get_detailed_list_restore_delete | ceph                                  |
```

Comment:
Do not understand this fully. Do you mean it is not supported by ceph or it needs ceph like back-end to do it?